index.js

```diff
@@ -110,8 +110,13 @@ function serveIndex(root, options) {
     // parse URLs
     var url = parseUrl(req);
     var originalUrl = parseUrl.original(req);
-    var dir = decodeURIComponent(url.pathname);
-    var originalDir = decodeURIComponent(originalUrl.pathname);
+
+    try {
```

Comment:
Can the try catch be moved out of this main function? Many of the supported Node.js versions for this module will deopt the entire function to slow due to the try catch existing.

I can do it too during merge if not adjusted prior to merge, so there is no rush, just makes merge easier when I get there :+1:

Reply:
I had some time today and remembered this PR, so updated the PR to move the `try` statement since it hadn't been done already.